scripts/secnetperf-helpers.psm1

```diff
@@ -199,7 +199,11 @@ function Invoke-Secnetperf {
             Write-Host $rawOutput
             if ($metric -eq "latency") {
                 $latency_percentiles = '(?<=\d{1,3}(?:\.\d{1,2})?th: )\d+'
-                $values[$tcp] += [regex]::Matches($rawOutput, $latency_percentiles) | ForEach-Object {$_.Value}
+                $perc = [regex]::Matches($rawOutput, $latency_percentiles) | ForEach-Object {$_.Value}
+                $values[$tcp] += $perc
```

Comment:
Why would splitting across two different lines make a difference?

Reply:
Oh I am using this to debug, to check if powershell is doing something weird where it truncates certain values. Once a bug has been found/fixed I can convert this PR from draft mode.